terraform/templates/snapshot-streaming.service

```diff
@@ -4,7 +4,7 @@ Description=Constellation Snapshot-Streaming
 [Service]
 User=ec2-user
 WorkingDirectory=/home/ec2-user/snapshot-streaming
-ExecStart=/home/ec2-user/snapshot-streaming/start
+ExecStart=/home/ec2-user/snapshot-streaming/start.sh
```

Comment:
In `terraform/ec2.tf` in line 98 the name of the script is provided without the `.sh` extension.
```
sudo chmod 774 /home/ec2-user/snapshot-streaming/start
```